api/KS.FiksProtokollValidator.WebAPI/TestCases/no.ks.fiks.politisk.behandling.klient.v1/HentUtvalgN1/testInformation.json

```diff
@@ -0,0 +1,38 @@
+﻿{
+    "testName": "Hent Utvalg",
+    "description": "Hent Utvalg",
+    "testStep": "Normalsituasjon 1 Input Hent Utvalg, retur Utvalg",
+    "messageType": "no.ks.fiks.politisk.behandling.klient.hentutvalg.v1",
+    "operation": "HentUtvalg",
+    "situation": "N1",
+    "expectedResult": "Alle Utvalg",
+    "queriesWithExpectedValues": [
+        {
+            "payloadQuery": "$",
+            "expectedValue": "utvalg",
+            "valueType": 1 // 0 = Verdi, 1 = Attributt
+        },
+        {
+            "payloadQuery": "$.utvalg",
+            "expectedValue": "utvalgId",
+            "valueType": 1 // 0 = Verdi, 1 = Attributt
+        },
+        {
+            "payloadQuery": "$.utvalg",
+            "expectedValue": "utvalgNavn",
+            "valueType": 1 // 0 = Verdi, 1 = Attributt
+        },
+        {
+            "payloadQuery": "$.utvalg[*].utvalgId",
+            "expectedValue": "*", // * = en hvilken som helst verdi som ikke er tom, null eller whitespace
+            "valueType": 0 // 0 = Verdi, 1 = Attributt
+        },
+        {
+            "payloadQuery": "$.utvalg[*].utvalgNavn",
+            "expectedValue": "*", // * = en hvilken som helst verdi som ikke er tom, null eller whitespace
+            "valueType": 0 // 0 = Verdi, 1 = Attributt
+        }
+    ],
+    "supported": true,
+    "protocol": "no.ks.fiks.politisk.behandling.klient.v1"
```

Comment:
Vi oppdaget at alle json filer kom med ved bygg + publish til output folder men ikke vedlegg (pdf osv.) og xml-filer.
De måtte markeres som "content" og sørge for at det ble "CopyToOutputFolder" i csproj-fil. Ellers vil ikke disse filene følge med output og mangle når det kjører på server. Se gjerne Dockerfile for hvordan det bygges.
Enkleste måten å sjekke om filene kommer med er å kjøre tilsvarende bygg lokalt og sjekke output folder. Jeg tror ikke vedtak.docx vil følge med as default.

Reply:
Jeg har fikset dette for de nye mappene!